torch/csrc/jit/register_special_ops.cpp

```diff
@@ -286,159 +286,166 @@ RegisterOperators reg({
-             }
-
-             auto default_type =
-                 at::typeMetaToScalarType(at::get_default_dtype());
-
-             if (dtype.isNone() && tensor.scalar_type() != default_type &&
-                 tensor.numel() == 0) {
-               AT_WARN(
-                   "Creating a tensor from an empty ",
-                   elem_type->str(),
-                   "list will create a tensor of default floating point type  (currently ",
-                   default_type,
-                   ") in python but a tensor of type ",
-                   elem_type->str(),
-                   " in torchscript.\n",
-                   "Pass in a dtype argument to ensure consistent behavior");
-             }
-             tensor.set_requires_grad(requires_grad);
-             push(stack, tensor);
-             return 0;
-           };
-         }),
-     Operator(
-         "aten::_assert_int_or_pair(int[] vals, str name, str message) -> Tensor",
-         [](const Node* node) {
-           return [](Stack& stack) {
-             // Everything is a list at the point this is used, so don't do
-             // anything
-             drop(stack, 3);
-             return 0;
-           };
-         }),
-     Operator(
-         "aten::_pack_sequence(Tensor output, Tensor batch_sizes, Tensor? sorted_indices, "
-         "Tensor? unsorted_indices) -> (Tensor, Tensor, Tensor?, Tensor?)",
-         [](Stack& stack) { return 0; }),
-     Operator("aten::_no_grad_uniform_(Tensor(a!) tensor, float a, float b) -> Tensor(a!)", [](Stack& stack) {
-       // TODO: remove when script supports setting grad mode
-       torch::NoGradGuard no_grad;
-
-       at::Tensor tensor;
-       double a;
-       double b;
-       pop(stack, tensor, a, b);
-       push(stack, tensor.uniform_(a, b));
-       return 0;
-     }),
-     Operator("aten::_no_grad_normal_(Tensor(a!) tensor, float mean, float std) -> Tensor(a!)", [](Stack& stack) {
-       // TODO: remove when script supports setting grad mode
-       torch::NoGradGuard no_grad;
-
-       at::Tensor tensor;
-       double mean;
-       double std;
-       pop(stack, tensor, mean, std);
-       push(stack, tensor.normal_(mean, std));
-       return 0;
-     }),
-     Operator("aten::_no_grad_fill_(Tensor(a!) tensor, float val) -> Tensor(a!)", [](Stack& stack) {
-       // TODO: remove when script supports setting grad mode
-       torch::NoGradGuard no_grad;
-
-       at::Tensor tensor;
-       double val;
-       pop(stack, tensor, val);
-       push(stack, at::fill_(tensor, val));
-       return 0;
-     }),
-     Operator("aten::_no_grad_zero_(Tensor(a!) tensor) -> Tensor(a!)", [](Stack& stack) {
-       // TODO: remove when script supports setting grad mode
-       torch::NoGradGuard no_grad;
-
-       at::Tensor tensor;
-       pop(stack, tensor);
-       push(stack, at::zero_(tensor));
-       return 0;
-     }),
-
-    });
+    DEFINE_TORCH_TENSOR_OP(float, double, at::scalar_to_tensor(scalar_val))
```

Comment:
What happened in this block?

Reply:
I think it was a bad rebase, this isn't changed in the final commit.